test/unit/client-side-encryption/requirements.helper.ts

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line @typescript-eslint/no-restricted-imports
+import { getAwsCredentialProvider, getGcpMetadata } from '../../../src/deps';
+
+// Data Key Stuff
+export const AWS_ACCESS_KEY_ID = process.env.AWS_ACCESS_KEY_ID;
+export const AWS_SECRET_ACCESS_KEY = process.env.AWS_SECRET_ACCESS_KEY;
+export const AWS_REGION = process.env.AWS_REGION;
+export const AWS_CMK_ID = process.env.AWS_CMK_ID;
+
+export const awsKmsProviders = {
+  aws: { accessKeyId: AWS_ACCESS_KEY_ID, secretAccessKey: AWS_SECRET_ACCESS_KEY }
+};
+export const awsDataKeyOptions = { masterKey: { key: AWS_CMK_ID, region: AWS_REGION } };
+
+export const SKIP_AWS_TESTS = [
```

Comment:
Can we integrate this logic into our typical testing env info? or are there more clean up tickets on the way w.r.t tests? I'm imagining these could be methods/properties on `TestConfiguration`.

Reply:
Yeah, there are all the remaining tests to be converted to TS. I wasn't planning on cleaning this up too much - these are unit tests, not integration, so we don't have our usual test requirement infrastructure.